front/app/containers/Admin/projects/project/nativeSurvey/FormResults/analysis/AnalysisInsights.tsx

```diff
@@ -22,32 +27,103 @@ type AnalysisInsight = {
   };
 };
 
+// Convert all values in the filters object to strings
+// This is necessary because the filters are passed as query params
+const convertFilterValuesToString = (filters?: IInputsFilterParams) => {
+  return (
+    filters &&
+    Object.entries(filters).reduce((acc, [key, value]) => {
+      return {
+        ...acc,
+        [key]: JSON.stringify(value),
+      };
+    }, {})
+  );
+};
+
 const Summary = ({
   summaryId,
   analysisId,
 }: {
   summaryId: string;
   analysisId: string;
 }) => {
+  const { formatMessage, formatDate } = useIntl();
   const { projectId, phaseId } = useParams() as {
     projectId: string;
     phaseId: string;
   };
   const { data } = useAnalysisSummary({ analysisId, id: summaryId });
 
   const summary = data?.data.attributes.summary;
+  const filters = data?.data.attributes.filters;
+  const accuracy = data?.data.attributes.accuracy;
+  const generatedAt = data?.data.attributes.created_at;
+
   if (!summary) {
     return null;
   }
+
   return (
-    <Text fontSize="s" mt="0px">
-      {replaceIdRefsWithLinks({
-        insight: summary,
-        analysisId,
-        projectId,
-        phaseId,
-      })}
-    </Text>
+    <Box
+      display="flex"
+      flexDirection="column"
+      justifyContent="space-between"
+      h="460px"
+      gap="16px"
+    >
+      <Box overflowY="auto" h="100%">
```

Comment:
I wonder which part specifies the scrollbar on the summary text. Is it `overflowY="auto"`?

Reply:
Yes, exactly!